pkg/reconciler/pipelinerun/timeout_test.go

```diff
@@ -42,14 +42,14 @@ func TestTimeoutPipelineRun(t *testing.T) {
 		wantErr               bool
 	}{{
 		name:           "no-resolved-taskrun",
-		embeddedStatus: config.DefaultEmbeddedStatus,
+		embeddedStatus: config.FullEmbeddedStatus,
```

Comment:
Can I confirm is this purposed to be DefaultEmbeddedStatus test case here or can I safely change to FullEmbeddedStatus? cc @abayer

Reply:
You will have to change test cases here to use ChildReferences instead of `TaskRuns/Runs` eventually in any case to proceed TEP-0100. Therefore, IMHO, it could be either in this PR or a separate PR.

Reply:
If it's not done in this PR, could you create an issue for it and link it here please? 🙏

Reply:
Thanks @afrittoli , updated and tracked at https://github.com/tektoncd/pipeline/issues/4954#issuecomment-1398431876. (I think in this PR we have to cover those test cases as `full` and `both` values are still supported at the moment)